components/Title.jsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
```

Comment:
This line isn't needed in Next because every page automatically uploads React

Reply:
Yep, mb